amplify/backend/function/publishopportunitylisting/src/api.d.ts

```diff
@@ -0,0 +1,40 @@
+interface ListingEvent {
+    id: string;
+    opportunityId: string;
+    name: string;
+    description?: string;
+    funders: string[];
+    openDate?: string;
+    closeDate?: string;
+    lastPublished?: string;
+}
+
+interface Opportunity {
+    id: string;
+    name: string;
+    description: string;
+    funders: string[];
+    fundersComplete?: boolean;
+    openDate?: string;
+    closeDate?: string;
+    websiteListings?: WebsiteListing[];
+    teammembers?: [TeamMember];
+    teammembersComplete?: boolean;
+    // Not sure what this is ... yet
+    type?: any;
```

Comment:
Change from "any" as we know the type now

Reply:
Fixed a few things here